src/apis/btc-core.ts

```diff
@@ -165,4 +192,44 @@ export class DefaultBTCCoreAPI implements BTCCoreAPI {
     getData<T>(response: Promise<AxiosResponse<T>>): Promise<T> {
         return response.then((v) => v.data);
     }
+
+    async broadcastOpReturnTx(
+        receiver: string,
+        amount: string,
+        data: string
+    ): Promise<{
+        txid: string;
+        rawTx: string;
+    }> {
+        if (!this.client) {
+            throw new Error("Client needs to be initialized before usage");
+        }
+        const paidOutput = {} as any;
+        paidOutput[receiver] = amount;
+        const raw = await this.client.command("createrawtransaction", [], [{ data: data }, paidOutput]);
+        const funded = await this.client.command("fundrawtransaction", raw);
+        const signed = await this.client.command("signrawtransactionwithwallet", funded.hex);
+        const response = await this.broadcastRawTransaction(signed.hex);
+        await delay(1000);
```

Comment:
What are these delays used for? I guess on regtest you would need to mine a block right after to have the tx included. If there is good reason for this, please add a comment